.circleci/config.yml

```diff
@@ -32,6 +36,7 @@ python_test_steps: &python_test_steps
       dockerize -wait tcp://localhost:5432 \
                 -wait tcp://localhost:21050 \
                 -wait tcp://localhost:50070 \
+                -wait tcp://localhost:9000 \
```

Comment:
I have a PR up to add the `yandex/clickhouse-server` image to our CI, which I'll merge soon (#1144). After that's merged you can remove everything that's currently here except the environment variables above.